### DISCUSSION TITLE
GE meetings July-September

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#136] [General]
@delph-in/participants especially those who are interested in regular meetings:

I think it would be great to continue meeting over the summer, hopefully every other week, more or less. 

Possible hours are American morning/European evening. I'd say the only realistic possibilities are 15, 16, or 17 UTC. So we need to pick a day so as to make it more likely that at least some of us will consistently join, even if different subsets. 

I created a when2meet: https://www.when2meet.com/?20499737-Hjlvw

The days there are July 10 through 15 but that's not important; the idea is just to pick a regular time that would at least sometimes work over the summer. So, treat "July 10" to just mean "some of the even Mondays". 

Thanks!

**Comment by olzama** [comment]
Only three people (including me) filled this out, and it looks like Thursdays 16:00 UTC might work for us. I know this week is ACL but I still scheduled a meeting for Thursday July 13, just in case someone who is not at ACL can make it! 

@danflick @arademaker @fcbond if you forgot to fill out the when2meet and the proposed time doesn't work for you at all, please let me know!

Otherwise, we can try to start meeting on Thursdays, starting this coming Thursday 16:00 UTC: 

https://zoom.us/j/93006336980?pwd=djc2UDJUYlNOTGM5eWRMWjdPQS9HUT09
Meeting ID: 930 0633 6980
Passcode: 184739

**Comment by danflick** [comment]
Thursdays will be fine for me, except not this first one. I'll be back home
for the next one.

Dan

**Comment by arademaker** [comment]
I just filled out the form. During the next two weeks, I can make it almost every day since FGV is in the middle year break and an I having my holidays from IBM. Starting August I can make Mon, Wed, or Fri afternoon.

**Reply by olzama** [reply]
I think we will maybe realistically only have a couple meetings between now and the start of the semester. The in September, we will have to change the schedule again. Let's keep Thursdays 16:00 UTC for this summer then!

**Comment by emilymbender** [comment]
Hi Olga,

Can you send out a calendar invite with the times & the zoom link embedded?

Thank you,
Emily

**Comment by olzama** [comment]
Sorry @emilymbender I just saw this message. Did you mean, send it to you or to those who typically attend, or?.. The idea is that maybe other @delph-in/participants who don't normally come to these will sometimes join, that's why I've been using the github board. I'll send an invite to the usual crowd I guess.

**Reply by emilymbender** [reply]
Thanks, Olga! It's good to use the github board, but I find the calendar invite very helpful, too! (Among other things, it gets onto my calendar reliably and handles the timezones...)

**Comment by olzama** [comment]
We will try to have a meeting today, 16:00 UTC:

https://zoom.us/j/93006336980?pwd=djc2UDJUYlNOTGM5eWRMWjdPQS9HUT09
Meeting ID: 930 0633 6980
Passcode: 184739

Then there are a couple more between now and the new semester.
